### PR TITLE
Refactor MTE-98 [v107] Remove unused code from FxScreenGraph

### PR DESCRIPTION
**Pull request by clarmso**
We would like to remove some unused code in FxScreenGraph. Over time, the UI changes on the browser makes some lines no longer be used.
https://mozilla-hub.atlassian.net/browse/MTE-98

### CODE REVIEW

**Review by isabelrios — APPROVED** [review]
LGTM! Thank you 
You could also removed what we talked on slack: https://github.com/mozilla-mobile/firefox-ios/blob/6a1dc66abcde7a35b64f3c6b039bd98a2fb8145d/Tests/XCUITests/FxScreenGraph.swift#L20
and the method that uses it that is not used by any test (as far as I have seen)

File: Tests/XCUITests/PhotonActionSheetTest.swift [hunk 1]
```diff
@@ -111,21 +111,6 @@ class PhotonActionSheetTest: BaseTestCase {
         waitForExistence(app.navigationBars["ShareTo.ShareView"], timeout: 10)
     }
 
-    private func disableFennec() {
-        navigator.nowAt(BrowserTab)
-        navigator.goto(PageOptionsMenu)
```
Comment (clarmso) [inline Tests/XCUITests/PhotonActionSheetTest.swift:116]: This is only place that `PageOptionsMenu` is used.

**Comment by clarmso** [comment]
@isabelrios After I removed `PageOptionsMenu`, the Smoketest2 failed because of the following:
```
/Users/vagrant/git/Tests/XCUITests/TopTabsTest.swift:409 - Expect predicate exists == true for "Switch" Button
```
But Bitrise indicated that all tests are passing. 😕 Should I just restart the tests on Bitrise?

**Comment by isabelrios** [comment]
> @isabelrios After I removed `PageOptionsMenu`, the Smoketest2 failed because of the following:
> 
> ```
> /Users/vagrant/git/Tests/XCUITests/TopTabsTest.swift:409 - Expect predicate exists == true for "Switch" Button
> ```
> 
> But Bitrise indicated that all tests are passing. 😕 Should I just restart the tests on Bitrise?

It is a flaky test. It failed once, in the re-run it passed, that's why bitrise is green. It is safe to land this 🚀

**Pull request merged**